sdk/Lusid.Sdk.Tests/tutorials/Ibor/StructuredResultStore.cs

```diff
@@ -0,0 +1,946 @@
+            string documentScope = "document-scope" + Guid.NewGuid();
+            // Will be considering documents with multiple portfolios and querying the data on grouped level
+            string resultType = "UnitResult/Grouped";
+            DataMapKey dataMapKey = new DataMapKey("1.0.0", "test-code");
+            DateTimeOffset effectiveAt = new DateTimeOffset(2022, 01, 19, 0, 0, 0, 0, TimeSpan.Zero);
+            // Will be creating two portfolios
+            string scope1 = "scope-" + Guid.NewGuid();
+            string portfolioCode1 = "pf-1";
+            string scope2 = "scope-" + Guid.NewGuid();
+            string portfolioCode2 = "pf-2";
+            
+            
+            // Create portfolios
+            var portfolioRequest1 = new CreateTransactionPortfolioRequest(
+                code: portfolioCode1,
+                displayName: $"Portfolio-{portfolioCode1}",
+                baseCurrency: "USD",
+                created: effectiveAt
+            );
+            _transactionPortfoliosApi.CreatePortfolio(scope1, portfolioRequest1);
+            var portfolioRequest2 = new CreateTransactionPortfolioRequest(
+                code: portfolioCode2,
+                displayName: $"Portfolio-{portfolioCode2}",
+                baseCurrency: "USD",
+                created: effectiveAt
+            );
+            _transactionPortfoliosApi.CreatePortfolio(scope2, portfolioRequest2);
+
+            // Create and upsert two instruments
+            string instrumentName1 = "AClientName1";
+            string instrumentName2 = "AClientName2";
+            string clientInstId1 = "clientInstId1";
+            string clientInstId2 = "clientInstId2";
+            var instruments = new Dictionary<string, InstrumentDefinition>
+            {
+                {
+                    "an-inst1", new InstrumentDefinition(instrumentName1,
+                        new Dictionary<string, InstrumentIdValue>
+                        {{
+                            "ClientInternal", new InstrumentIdValue(clientInstId1)
+                        }}
+                    )
+                },
+                {
+                    "an-inst2", new InstrumentDefinition(instrumentName2,
+                        new Dictionary<string, InstrumentIdValue>
+                        {{
+                            "ClientInternal", new InstrumentIdValue(clientInstId2)
+                        }}
+                    )
+                },
+            };
+            var upsertResponse = _instrumentsApi.UpsertInstruments(instruments);
+            
+            // Retrive LusidInstrumentId's of the upserted instruments
+            List<string> luids = upsertResponse.Values
+                .Select(inst => inst.Value.LusidInstrumentId)
+                .ToList();
+            
+            // Create and upsert two transactions, one per portfolio.
+            var transactionRequest1 = new List<TransactionRequest>
```

Comment:
Why do we need to put in transactions?

Reply:
Wanting to see the results per transaction (two per transaction as they are of type "Buy"), if there are no transactions we obtain nothing back